capstone-backend/src/main/java/com/google/sps/servlets/SearchServlet.java

```diff
@@ -46,7 +46,6 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-
```

Comment:
nit: Should probably be in a separate formatting PR. Have you looked into setting up an auto-formatter?

Reply:
I added local testing lines and accidentally removed the starting line of the Servlet files when I removed them to commit this code.
Fixed.

The formatter we use is Gitpod, but it requires clicking a formatting button and only handles formatting code, it does not deal with extra whitespace like this. If we had the time, I think it would have been a good idea to implement some sort of linter at the beginning of the project to ensure consistency but at this point there is not sufficient time.